run_local.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+println () {
+  printf '%*s\n' "${COLUMNS:-$(tput cols)}" '' | tr ' ' -
+  echo $1
+  printf '%*s\n' "${COLUMNS:-$(tput cols)}" '' | tr ' ' -
+  }
+
+healthyStatus='"sqs": "running"'
+
+println "BUILDING PROJECT"
+./gradlew --no-daemon assemble
+
```

Comment:
very minor, but wouldn't the docker compose below do a build anyway since the compose file has a `build` element ? (and btw I'm not sure)

Reply:
You are correct it does/should. I have a strange issue locally in which the docker compose seems to cache use a cached version of the jar. 

The other solution would be to use this `docker-compose build --no-cache` to ensure the cache isn't used, but its really slow

Reply:
Btw, I don't know why rebuilding manually works, however it seems to